bin/command.php

```diff
@@ -382,15 +382,15 @@ private static function gen_cmd_pages( $cmd, $parent = array(), $verbose = false
 			$docs = $cmd['longdesc'];
 			$docs = htmlspecialchars( $docs, ENT_COMPAT, 'UTF-8' );
 
-			// decrease header level
+			// decrease header level.
```

Comment:
While we're at it, we can also capitalize the first word of the comments to comply with WPCS.
```suggestion
			// Decrease header level.
```